core/src/main/java/com/sequenceiq/cloudbreak/controller/validation/VolumeCountValidator.java

```diff
@@ -46,7 +45,7 @@ private boolean isAwsTemplate(TemplateRequest value) {
     }
 
     private boolean isEphemeralVolume(TemplateRequest value) {
-        return AwsVolumeType.Ephemeral.name().equalsIgnoreCase((String) value.getParameters().get(AwsTemplateParam.VOLUME_TYPE.getName()));
+        return "ephemeral".equalsIgnoreCase((String) value.getParameters().get(AwsTemplateParam.VOLUME_TYPE.getName()));
```

Comment:
Can these strings be extracted to constants so they won't have to be changed in several different places when needed?